Controller/ProfileManager.php

```diff
@@ -36,29 +37,28 @@ class ProfileManager
      * @param FormFactory $formFactory
      */
     public function __construct(
-        UserDiscriminator $userDiscriminator,
-        ContainerInterface $container,
-        ProfileController $controller,
-        FormFactory $formFactory
-    ) {
+    UserDiscriminator $userDiscriminator, ContainerInterface $container,
```

Comment:
This change is useless and against PSR-2. Please restore previous line splitting.